### DISCUSSION TITLE
article/hip-hop-the-songs-that-shook-america

### DISCUSSION THREAD

**Discussion by giscus** [thedavidweng/NotionNext#2] [Announcements]
# article/hip-hop-the-songs-that-shook-america

《嘻哈: 震撼美国的歌曲》系列纪录片的每一集都聚焦于一首对美国音乐和文化进化至关重要的开创性歌曲

https://blog.davidweng.tk/article/hip-hop-the-songs-that-shook-america

**Comment by thedavidweng** [comment]
如果有人对文章发布表情或者评论，Giscus 的 BOT 会自动在 Discussions 中创建对应的 Announcement
